Makefile

```diff
@@ -23,7 +33,8 @@ db-stop: ## Shut down the DB container
 
 ##@ [Application]
 configure: ## Configures the application when setting it for the first time
-	make install \
+	[ -f .env ] || cp .env.example .env \
```

Comment:
This should be in the build command instead because it doesn't make sense to run the configure before the build since the make install and make art commands need the images built to run

Reply:
This is where the problem occurs, the build being executed before configure will present an error because the `docker/app/Dockerfile` mentions:

`COPY .env .env`


Soon the error will occur because the file does not exist yet, hence the inversion of the order as well as the adjustment to verify whether or not the file exists before copying the .env.example

Reply:
But the make configure needs the images built. I don’t understand what’s the problem here since it already copies the file in the build

Reply:
The make build command first creates the .env and then run the build command with Docker. IDK why this would fail when trying to run the build command first

Reply:
Okay, really, my mistake.

I'll just add the validation, to copy if it doesn't exist.